src/header/common/origin.rs

```diff
@@ -0,0 +1,29 @@
+header! {
+    /// 'Origin' request header,
+    /// part of [CORS](http://www.w3.org/TR/cors/#origin-request-header)
```

Comment:
We should reference http://tools.ietf.org/html/rfc6454#section-7 instead and ensure that this header implements that specification.